Index: bot.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import datetime\r\n\r\nfrom aiogram.utils import executor\r\nfrom create_bot import dp, scheduler\r\nfrom db import db\r\nfrom handlers.other import on_startup\r\nimport logging\r\nlogging.basicConfig()\r\nlogging.getLogger('apscheduler').setLevel(logging.DEBUG)\r\n\r\n\r\nfrom handlers import admin, client\r\n\r\n\r\nasync def send_messages(forecast_id):\r\n    print(forecast_id)\r\n    # tg_ids = db.get_all_tg_id()\r\n    # forecast = db.get_forecast(forecast_id)\r\n    # for tg_id in tg_ids:\r\n    #     await bot.send_message(tg_id, forecast[1])\r\n    #     await asyncio.sleep(5)\r\n\r\n\r\nadmin.register_handlers_admin(dp)\r\nclient.register_handlers_client(dp)\r\nexecutor.start_polling(dp, on_startup=on_startup())\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/bot.py b/bot.py
--- a/bot.py	
+++ b/bot.py	
@@ -23,5 +23,3 @@
 
 admin.register_handlers_admin(dp)
 client.register_handlers_client(dp)
-executor.start_polling(dp, on_startup=on_startup())
-
